scripts/build-media-variants.js

```diff
@@ -76,11 +76,16 @@ const targetClassArray = [
   'z4',
   'z5',
   'mx-auto',
-  'viewport-third',
-  'viewport-half',
-  'viewport-twothirds',
-  'viewport-almost',
-  'viewport-full',
+  'h-viewport-1\\/3',
+  'h-viewport-1\\/2',
+  'h-viewport-2\\/3',
+  'h-viewport-9\\/10',
+  'h-viewport-full',
+  'hmax-viewport-1\\/3',
+  'hmax-viewport-1\\/2',
+  'hmax-viewport-2\\/3',
+  'hmax-viewport-9\\/10',
```

Comment:
Thoughts on keeping this `-almost`? `9/10` is a weird fraction and `almost` has some nice semantic meaning.

Reply:
`almost` never felt right to me. 9/10 accurately describes this class using similar patterns in place elsewhere in assembly.

Reply:
Unless all fractions are there and standard across our usage of fractions, `9/10` just feels forgettable and you are fitting it into a self-inflicted system.

> If we extend the scale to add more stops, 9/10 will always mean 9/10

I highly doubt we will extend the scale for `vh` units.

---

Another alternative: Can we just drop it? I did a quick audit of its usage and it can quite easily be dropped. It existed for splash images where you wanted a little reveal to indicate there was more below the fold. This seems like a fine candidate as an inline rule.